src/store/user/index.ts

```diff
@@ -38,7 +38,10 @@ const initialState: UserState = {
   errors: initialUserErrors,
   touched: initialUserTouched,
   isLoading: true,
-  isSubmitted: false
+  isSubmitted: false,
+  login: '',
```

Comment:
actually, login, email and name should be there. I'll fix it later, but you can try to investigate an issue to improve you TS skill